substratools/metrics.py

```diff
@@ -153,12 +124,12 @@ def score(self, fake_data=False, n_fake_samples=None):
 
         logger.info("launching scoring task")
 
-        inputs = {InputIdentifiers.y: y, InputIdentifiers.predictions: y_pred_path}
```

Comment:
what is the benefice of removing `InputIdentifiers` and `OutputIdentifiers`?

Reply:
None to be honest, as it was to be removed when the metric and the algo will be merge I took a step ahead. 
I can revert this if you want

Reply:
I would rather keep it, I expect it to be useful for some PRs yet

Reply:
I'll revert then

Reply:
I re add it within the test. Here it is only hard coded it doesn't make a lot of sense to have it to be honest. 
When those values were shared with the algo and the opener it did but now I think there is no reason to keep them.